### PR TITLE
Library Values

### PR DESCRIPTION
**Pull request by hayesgm**
This patch uses library values for a variety of constants as a way to clean up a variety of copy-pasted code in our QuarkScripts code. Small clean-up.

### CODE REVIEW

File: test/quark-core/QuarkWallet.t.sol [hunk 1]
```diff
@@ -235,7 +235,6 @@ contract QuarkWalletTest is Test {
         // gas: do not meter set-up
         vm.pauseGasMetering();
 
-        Counter counter = new Counter();
         assertEq(counter.number(), 0);
```
Comment (fluffywaffles) [inline test/quark-core/QuarkWallet.t.sol:238]: so were we... re-initializing the counter here? :laughing:

**Review by fluffywaffles — APPROVED** [review]
:sunglasses:

File: src/quark-core/src/QuarkWallet.sol [hunk 2]
```diff
@@ -112,17 +125,16 @@ contract QuarkWallet is IERC1271 {
...
 
-    /// @notice Well-known --
-    bytes32 public constant ACTIVE_NONCE_SLOT = bytes32(uint256(keccak256("quark.v1.active.nonce")) - 1);
+    /// @notice Well-known storage slot for the nonce of the script that's currently executing.
+    bytes32 public constant ACTIVE_NONCE_SLOT = QuarkWalletMetadata.ACTIVE_NONCE_SLOT;
 
-    /// @notice Well-known --
-    bytes32 public constant ACTIVE_SUBMISSION_TOKEN_SLOT =
-        bytes32(uint256(keccak256("quark.v1.active.submissionToken")) - 1);
+    /// @notice Well-known storage slot for the submission token of the script thatg's currently executing.
```
Comment (kevincheng96) [inline src/quark-core/src/QuarkWallet.sol:136]:
```suggestion
    /// @notice Well-known storage slot for the submission token of the script that's currently executing.
```

**Review by kevincheng96 — APPROVED** [review]

File: src/quark-core/src/QuarkWallet.sol (hunk 2 above)
Comment (hayesgm) [reply src/quark-core/src/QuarkWallet.sol]: thanks

**Pull request merged**